ReadThread.py

```diff
@@ -0,0 +1,76 @@
+from typing import Dict
+
+from PyQt5 import QtCore
+from PyQt5.QtCore import pyqtSignal
+import queue
+
+import RadioController
+
+class ReadThread(QtCore.QThread):
+    # sig_received = pyqtSignal(list)  # TODO Review this change with Andrei
+    sig_received = pyqtSignal(object)  # TODO is it safe to generalize
```

Comment:
A bit of an artifact but useful to ask now especially with this change: Is it reasonable to have this generalization? It was necessary to change it for my earlier changes due to the way that data is now being sent back as dictionary but not a list anymore.

Reply:
Just looked into it a bit. That should be fine.